### PR TITLE
Add starter style guides

### PR DESCRIPTION
**Pull request by tomquisel**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,6 +1,26 @@
...
+
+### Git Commit Messages
+
+* Use the present tense ("Add feature" not "Added feature")
+* Use the imperative mood ("Move cursor to..." not "Moves cursor to...")
+
+### Pull Requests
+
+* PR titles start with package name, ex. “horizon/ingest: fix...”
+* PRs must update the CHANGELOG with a small description of the change
```
Comment (ire-and-curses) [inline CONTRIBUTING.md]: Could link to Changelog here

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,6 +1,26 @@
...
+* PRs are merged into master or release branch using squash merge
+* Carefully think about where your PR fits according to [semver](https://semver.org). Target it at master if it’s only a patch change, otherwise if it contains breaking change or significant feature additions, set the base branch to the next major or minor release.
+* Keep PR scope narrow. Expectation: 20 minutes to review max
+* Explicitly differentiate refactoring PRs and feature PRs. Refactoring PRs don’t change functionality. They usually touch a lot more code, and are reviewed in less detail. Avoid refactoring in feature PRs.
+
+### Go Style Guide
+
+* Use `gofmt` or preferably `goimports` to format code
```
Comment (ire-and-curses) [inline CONTRIBUTING.md]: I think they do different things - should use both.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,6 +1,26 @@
...
 
-Then please [sign the Contributor License Agreement](https://docs.google.com/forms/d/1g7EF6PERciwn7zfmfke5Sir2n10yddGGSXyZsq98tVY/viewform?usp=send_form).
+Check out the [Stellar Contribution Guide](https://github.com/stellar/docs/blob/master/CONTRIBUTING.md) for details.
 
+## Style guides
+
+### Git Commit Messages
+
+* Use the present tense ("Add feature" not "Added feature")
+* Use the imperative mood ("Move cursor to..." not "Moves cursor to...")
```
Comment (ire-and-curses) [inline CONTRIBUTING.md]: mood -> mode

Comment (tomquisel) [reply CONTRIBUTING.md]: It's a real thing: https://en.wikipedia.org/wiki/Imperative_mood

File: CONTRIBUTING.md (hunk 2 above)
Comment (tomquisel) [reply CONTRIBUTING.md]: `goimports` does what `gofmt` does plus a little more: https://godoc.org/golang.org/x/tools/cmd/goimports so no reason to use both.

**Comment by tomquisel** [comment]
@ire-and-curses I'm just going to leave the convention thing in the README alone. It's a little different from a style guide, and it's a bit of a can of worms.

**Pull request merged**